src/components/input/input-base.ts

```diff
@@ -41,8 +40,7 @@ export abstract class IgcInputBaseComponent
   protected inputId = `input-${nextId++}`;
 
   /** The value attribute of the control. */
-  @blazorSuppress()
```

Comment:
Probably not a good idea to drop the suppress here btw.

Although, in the interest of cleaner API gen down the line I'm considering an alternative approach - exposing base properties as public only on the theoretical leaf nodes where the type would be fixed and keeping this one protected instead

Reply:
@onlyexeption This should still remain unchanged